frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -1995,19 +1934,22 @@ export async function getAgglomeratesForSegmentsFromDatastore<T extends number |
   mappingId: string,
   segmentIds: Array<T>,
 ): Promise<Mapping> {
+  const params = new URLSearchParams();
+
   const segmentIdBuffer = serializeProtoListOfLong<T>(segmentIds);
-  const listArrayBuffer: ArrayBuffer = await doWithToken((token) =>
-    Request.receiveArraybuffer(
-      `${dataStoreUrl}/data/datasets/${dataSourceId.owningOrganization}/${dataSourceId.directoryName}/layers/${layerName}/agglomerates/${mappingId}/agglomeratesForSegments?token=${token}`,
+  const listArrayBuffer: ArrayBuffer = await doWithToken((token) => {
+    params.append("token", token);
+    return Request.receiveArraybuffer(
+      `${dataStoreUrl}/data/datasets/${dataSourceId.owningOrganization}/${dataSourceId.directoryName}/layers/${layerName}/agglomerates/${mappingId}/agglomeratesForSegments?${params}`,
```

Comment:
Very nitpicky but what about

```suggestion
  const segmentIdBuffer = serializeProtoListOfLong<T>(segmentIds);
  const listArrayBuffer: ArrayBuffer = await doWithToken((token) => {
    const params = new URLSearchParams({token});
    return Request.receiveArraybuffer(
      `${dataStoreUrl}/data/datasets/${dataSourceId.owningOrganization}/${dataSourceId.directoryName}/layers/${layerName}/agglomerates/${mappingId}/agglomeratesForSegments?${params}`,
```